requirements_gpu.txt

```
@@ -0,0 +1 @@
+cupy
```

This was the package being installed before, unless you changed it intentionally.
```suggestion
cupy-cuda102
```

---

Good catch!